osu.Game/Skinning/PausableSkinnableSound.cs

```diff
@@ -41,7 +41,8 @@ private void load(ISamplePlaybackDisabler samplePlaybackDisabler)
                         // it's not easy to know if a sample has finished playing (to end).
                         // to keep things simple only resume playing looping samples.
                         else if (Looping)
-                            base.Play();
+                            // schedule so we don't start playing a sample which is no longer alive.
+                            Schedule(base.Play);
```

Comment:
Do we want this instead of an `IsAlive` check? Wouldn't it also require scheduling the above `base.Stop()` too?

For example, seek into the future, trigger valuechanged a few times by starting/stopping play, then seek back to the object. Wouldn't that make the object play the samples because the schedules would be then be run?

Reply:
The `IsAlive` thing is the reason this is as it is. As discussed yesterday `IsAlive` can't be used here (unless the `DrawableHitObject` is being passed in).

Will check on the schedule during rewinds, didn't notice an issue but I can understand why there is cause for concern.

Reply:
I've added a re-check of `RequestedPlaying` inside the schedule, which is enough to make this safe (the play is only issued to `Looping` samples, which need to be manually stopped, so it's guaranteed their `RequestedPlaying` will be in the correct state at the point of the scheduled callback being run).